sf_examples/megaverse_examples/megaverse/megaverse_utils.py

```diff
@@ -0,0 +1,109 @@
+import gym
+from megaverse.megaverse_env import MegaverseEnv, make_env_multitask
+
+from sample_factory.envs.env_utils import RewardShapingInterface, TrainingInfoInterface
+from sample_factory.utils.utils import log
+
+
+class MegaverseSpec:
+    def __init__(self, name):
+        self.name = name
+
+
+MEGAVERSE_ENVS = [
+    MegaverseSpec("megaverse_TowerBuilding"),
```

Comment:
Why not also have all the other specs?
Can you just copy them from Megaverse repo?

Reply:
Also, is there any reason to use `megaverse_` prefix for everything? Can we just do `TowerBuilding`?